src/t_stream.c

```diff
@@ -2313,8 +2457,8 @@ void streamDelConsumer(streamCG *cg, streamConsumer *consumer) {
  * Consumer groups commands
  * ----------------------------------------------------------------------- */
 
-/* XGROUP CREATE <key> <groupname> <id or $> [MKSTREAM]
- * XGROUP SETID <key> <groupname> <id or $>
+/* XGROUP CREATE <key> <groupname> <id or $> [MKSTREAM] [OFFSET offset]
+ * XGROUP SETID <key> <groupname> <id or $> [offset]
```

Comment:
the top comment of the PR doesn't list these command argument changes.

Reply:
Addressed.